bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala

```diff
@@ -41,7 +46,10 @@ sealed abstract class ITCRequest {
   def intParameter(name: String): Int = parameter(name).toInt
 
   /** Gets the named value as a double. */
-  def doubleParameter(name: String): Double = parameter(name).toDouble
+  def doubleParameter(name: String): Double = parameter(name).trim() match {
+    case ""   => 0.0
+    case s    => s.toDouble
+  }
```

Comment:
As I understand it, the idea is to clean up validation at some point to handle invalid input like "foo" here.  Why should `intParameter` not be given the same treatment as `doubleParameter` though?  Would it make sense to pass in the default value or is `0.0` always what we want?